src/pages/LandingPage.js

```diff
@@ -1,66 +1,66 @@
-import styled from 'styled-components';
-import { useNavigate } from 'react-router-dom';
+import NavAnimation from '../components/FramerMotion';
+
 import useStore from '../hooks/useStore';
-import backround from '../images/background.jpg';
+import styled from 'styled-components';
+import { NavLink } from 'react-router-dom';
 import { motion } from 'framer-motion';
-import NavAnimation from '../components/FramerMotion';
+
+import backround from '../images/background.jpg';
```

Comment:
```suggestion
import background from '../images/background.jpg';
```
Looks like you don't need the image?